buildenv/jenkins/common/build.groovy

```diff
@@ -155,8 +155,15 @@ def checkout_pullrequest() {
 
         // Setup PR_IDs for dependent changes
         for (DEPEND in DEPENDS_ARRAY) {
-            String REPO = DEPEND.substring(DEPEND.indexOf("/") + 1, DEPEND.indexOf("#"));
-            String PR_ID = DEPEND.substring(DEPEND.indexOf("#") + 1, DEPEND.length());
+            if (DEPEND.contains('http')) {
+                // eg htts://github.com/elclipse/omr/pull/123
```

Comment:
Perhaps we want `startsWith` instead of `contains`.
Is the typo in "elclipse" intentional?

Reply:
And the typo in `htts`?

Reply:
it's a comment :laughing: 
I will fix and change to `startsWith`